src/Disks/DiskType.h

```diff
@@ -17,7 +17,6 @@ enum class ObjectStorageType
 {
     None,
     S3,
-    S3_Plain,
```

Comment:
Are these enum values serialized anywhere?

Reply:
In zero copy replication, but it cannot be used with plain metadata type.